package.json

```diff
@@ -15,7 +15,9 @@
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
     "axios": "^1.3.4",
+    "bootstrap": "^5.2.3",
     "react": "^18.2.0",
+    "react-bootstrap": "^2.7.4",
```

Comment:
Waarom is dit toegevoegd? Gebruik je Bootstrap voor iets? In principe gebruiken we enkel MIUI components